pulp_container/app/json_schemas.py

```diff
@@ -78,13 +87,20 @@ def get_descriptor_schema(
             "type": "string",
             "enum": [MEDIA_TYPE.MANIFEST_OCI],
         },
-        "config": get_descriptor_schema(ALLOWED_ARTIFACT_TYPES),
+        "artifactType": {"type": "string"},
```

Comment:
sorry i was supposed to comment here: you forgot the subject! :D

Reply:
it is part of index and image manifest

Reply:
ahnn!! nice catch!